randovania/games/am2r/json_data/Main Caves.txt

```diff
@@ -1374,7 +1383,10 @@ Extra - minimap_data: [{'x': 53, 'y': 34}, {'x': 53, 'y': 35}, {'x': 53, 'y': 36
   * Layers: default
   * Open Passage to Mines Alpha Nest Access/Dock to Mines Intersection
   > Dock to Mining Facility Save Station
-      Hijump With Power Grip Wall
```

Comment:
Drill excavation is missing a way to leave right now when you start at the save station: you should make it possible to hit the crystal from the right (patcher should then ensure that if you start from the right, that the drill is gone so you can leave)

Reply:
Is the `Enabled Cutscene Skips` necessary for the drill to be moved or is this unnecessary?
![image](https://github.com/randovania/randovania/assets/125271738/9360af5b-caeb-4066-b7bf-8dda402c67eb)

Reply:
> patcher should then ensure that if you start from the right, that the drill is gone so you can leave

This was implying that cutscene skips should not be necessary for it.